enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -90,12 +90,18 @@ func sizeFn(value interface{}) int64 {
 }
 
 func evictFn(value interface{}, reason lru.EvictionReason) {
-	if v, ok := value.(*entry); ok {
-		syscall.Unlink(v.value)
-		if reason == lru.SizeEviction {
-			age := time.Since(time.UnixMicro(v.addedAtUsec)).Microseconds()
-			metrics.FileCacheLastEvictionAgeUsec.Set(float64(age))
-		}
+	// TODO(sluongng): consider using Go generic to avoid type assertion.
```

Comment:
I'm not sure this is the right place for this TODO - LRU is used pretty extensively in the codebase. I would omit this for now.

Reply:
Agree